packages/client/components/TeamPrompt/TeamPromptOptionsMenu.tsx

```diff
@@ -62,14 +65,17 @@ const TeamPromptOptionsMenu = (props: Props) => {
   const atmosphere = useAtmosphere()
   const {onCompleted, onError} = useMutationProps()
   const {history} = useRouter()
+  const isEnded = !!endedAt
   const hasRecurrenceEnabled = meetingSeries && !meetingSeries.cancelledAt
+  const hasActiveMetings = !!meetingSeries?.activeMeetings?.length
+  const canToggleRecurrence = isEnded ? hasActiveMetings : true
```

Comment:
+1 `!isEnded || hasActiveMeetings`?

Reply:
this won't allow ending the recurrence of ended meeting (!true || false). my condition was wrong as well, and as we use the same button for a start and stop, the condition to disable it is quite complex. 

the correct and a little verbose but easy-to-read version would look like that
```
  const canStartRecurrence = !isEnded
  const canEndRecurrence = !isEnded || !hasActiveMetings
  const canToggleRecurrence = hasRecurrenceEnabled ? canEndRecurrence : canStartRecurrence
```

wdyt?